index.html

```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
 </div>
   </section>
 
-  <div class="section-cards">
+  <div>
+    <div class="project section-cards">
```

Comment:
- [ ] On desktop, the "see live" button is misaligned.  Kindly correct it so it looks like the Figma:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/156446519-8325aa5a-b1a0-4f73-9fac-ac920d94e52d.png) | ![image](https://user-images.githubusercontent.com/57421823/156446546-1cda5533-6a2d-482f-a7e3-22bdc75509aa.png)

That way, your desktop popup will be consistent 💪